gateway/src/main/kotlin/Event.kt

```diff
@@ -491,6 +535,7 @@ public sealed class Event {
                     ),
                     sequence
                 )
+
```

Comment:
could you remove the line breaks again? i think `)` having their own lines is enough spacing between branches (maybe adjust your formatter)